public_src/components/modal/modal.component.ts

```diff
@@ -57,6 +70,39 @@ export class ModalComponent {
     }
   }
 
+  /**
+   * Saves ref correction
+   */
+  private saveRefTag(): void {
+    let refsForTag = [];
+    for (let rel of this.addedMissingSuggestionsRefs) {
+      refsForTag.push(rel.tags.ref);
+    }
+    for (let rel of this.addedFromNearbySuggestionsRefs) {
+      refsForTag.push(rel.tags.ref);
+    }
+    refsForTag = refsForTag.concat(this.newAddedRefs);
+    let refString: string = '';
+    refsForTag.forEach((item, index) => {
+      refString = (index !== refsForTag.length - 1) ? refString + item + ', ' : refString + item;
+    });
+    refString = this.errorObject.tags.route_ref + refString;
+    this.createChangeForRefTag(refString);
+    this.addToMembers(this.addedFromNearbySuggestionsRefs);
+    this.bsModalRef.hide();
+    let popUpElement = this.mapSrv.getPopUpFromArray(this.mapSrv.currentPopUpFeatureId);
+    MapService.addHoverListenersToPopUp(popUpElement);
+    this.mapSrv.popUpArr   = this.mapSrv.popUpArr.filter((popup) => popup['_leaflet_id'] !== this.mapSrv.currentPopUpFeatureId);
```

Comment:
I would prefer multiline `.filter( () => { return ... })` for consistency everywhere and smaller text width